lab-brianna/server.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const Client = require('./model/client');
+const net = require('net');
+const EE = require('events').EventEmitter;
+const ee = new EE();
+const server = module.export = net.createServer();
+const PORT = process.env.PORT || 4000;
+
+const pool = [];
+
+//default error command message
+ee.on('default', (client, string) => {
+  client.socket.write(`Not a valid command: ${string.split(' ', 1)}\n`);
+});
+
+//sends a message to all users
+ee.on('\\all', (client, string) => {
```

Comment:
So Im thinking you'll want to change these to '/command'. Then when you target it later on you'll only need to pass on the command.